### PR TITLE
Func pipeline removal

### PR DESCRIPTION
**Pull request by Lawreros**
Removed the functional pipeline files/functions from the pipeline. Also removed the "group" analysis option, which resulted in having to change the travis and setup.py files.

### CODE REVIEW

File: ndmg/scripts/ndmg_bids.py
Comment (loftusa) [inline ndmg/scripts/ndmg_bids.py:327]: Great job! Looks like it passes on travis. One minor change (that might be slightly nitpicky): 
if somebody tries to pass `--modality func`, could you have the pipeline raise `NotImplementedError`?
https://docs.python.org/3/library/exceptions.html#NotImplementedError

Other than that, I think we're good -- thanks!

Comment (Lawreros) [reply ndmg/scripts/ndmg_bids.py:327]: Good point, added it in and am waiting on travis

**Review by loftusa — APPROVED** [review]

**Comment by loftusa** [comment]
Approved! Feel free to merge once travis finishes.

**Comment by loftusa** [comment]
hold up on merging - would be good for @dPys to check as well

**Review by dPys — APPROVED** [review]

**Pull request merged**
